README.md

```diff
@@ -30,6 +30,7 @@ For any support regarding this plugin, please create a github issue.
 3. `cd pvr.waipu && mkdir build && cd build`
 4. `cmake -DADDONS_TO_BUILD=pvr.waipu -DADDON_SRC_PREFIX=../.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=../../xbmc/addons -DPACKAGE_ZIP=1 ../../xbmc/cmake/addons`
 5. `make`
+6. `cp -R ../../xbmc/addons/pvr.waipu/ ../../xbmc/kodi-build/addons`
```

Comment:
The make command puts the plugin into the source folder. But it seems to have to be put into the build folder.

Reply:
Instead of copying the files you can simply adjust the path after`-DCMAKE_INSTALL_PREFIX` to the target directory. E.g. `-DCMAKE_INSTALL_PREFIX ../../xbmc/kodi-build/addons`

Reply:
I've modified the cmake command to mention the Kodi build folder.
If you do not want to change the instructions just tell me, then I'll remove the part of the change.

For background: I started with your instructions for the build process, without building Kodi first.
When the Waipu plugin is already installed before running cmake on Kodi, it will cause errors about RapidJson dependency not found because cmake somehow wants to pick it up from the pvr.waipu dir.
This caused be some time to figure out.
Telling people to make sure to build Kodi first is helpful.
From your instructions it looked like Kodi will pick up the plugin automatically from the sources if installed there, but this also did not work for me. Hence the update to use a different install prefix.

Could still be that I am doing something completely wrong...

Reply:
Not sure if it is required to build kodi before. This issue never happend to me. But I usually have a self compiled version on my pc. I think adding the hint helps people having the same issue like you. So lets keep it.